src/utils/table-utils/data-container-utils.ts

```diff
@@ -21,18 +21,32 @@
 import {RowDataContainer} from './row-data-container';
 import {IndexedDataContainer} from './indexed-data-container';
 
-/** @type {import('./data-container-utils').DataForm} */
+import {DataContainerInterface} from './data-container-interface';
+import {Field} from './kepler-table';
+
+export type DataContainerOptions = {
+  inputDataFormat?: string; // one of DataForm
+  fields?: Field[];
+};
+
 export const DataForm = {
   ROWS_ARRAY: 'ROWS_ARRAY'
 };
 
-/** @type {import('./data-container-utils').DataContainerOptions} */
-const defaultOptions = {
+const defaultOptions: DataContainerOptions = {
   inputDataFormat: DataForm.ROWS_ARRAY
 };
 
-/** @type {typeof import('./data-container-utils').createDataContainer} */
-export function createDataContainer(data, options = defaultOptions) {
+/**
+ * Creates a data container wrapper for the data.
+ * @param data Data.
+ * @param options Options.
+ * @returns A data container object which is based on data and options.
+ */
+export function createDataContainer(
+  data: any[],
+  options: Partial<DataContainerOptions> = defaultOptions
```

Comment:
both `inputDataFormat` and `fields` are optional, this should just be `options: DataContainerOptions`